github/compliance/issue.py

```diff
@@ -66,7 +71,7 @@ def repository_labels(
     artifact: cm.Artifact | None,
     issue_type: str | None=_label_bdba,
     extra_labels: typing.Iterable[str]=None
-):
+) -> typing.Generator[str, None, None]:
```

Comment:
is `repository_labels` still used?

Reply:
yes, `repository_labels` yields _all_ labels for a repo, e.g. `cicd/auto-generated` AND the actual artifact digest label.

Reply:
I separated the digest calculation from repo label yielding as a preparation